classes/library.rb

```diff
@@ -0,0 +1,59 @@
+class Library
+  attr_accessor :books, :orders, :readers, :authors
+  attr_writer :data
+  def initialize(books = nil, orders = nil, readers = nil, authors = nil)
+    @books = books
+    @orders = orders
+    @readers = readers
+    @authors = authors
+  end
+
+  def most_popular_book
+    most_popular('book').title
+  end
+
+  def often_takes_books
+    most_popular('reader').name
+  end
+
+  def how_order_most_popular
+    @data.orders.uniq(&:reader).map(&:book).group_by(&:title).values.first(3).map(&:length).sum
```

Comment:
1. load data to collections
2. return readers who took one of three popular book